OpenRA.Game/FileFormats/Png.cs

```diff
@@ -66,7 +67,7 @@ public Png(Stream s)
 							Width = IPAddress.NetworkToHostOrder(ms.ReadInt32());
 							Height = IPAddress.NetworkToHostOrder(ms.ReadInt32());
 
-							var bitDepth = ms.ReadUInt8();
+							bitDepth = ms.ReadUInt8();
```

Comment:
i see no validation on the maximum of this value. unlikely that it is corrupt, but i imagine you don't support all formats?

Reply:
The `IsPaletted` method performs the validation. It throws for any unsupported formats.